src/components/hero/_hero.scss

```diff
@@ -62,4 +80,227 @@
             }
         }
     }
+    &--navy-blue & {
+        &__details {
+            color: var(--ons-color-text-inverse) !important;
+            padding: 2rem 0 6rem;
+            @include mq(l) {
+                padding: 4rem 0;
+            }
+        }
+
+        &__text {
+            padding: 0;
+        }
```

Comment:
I tried setting margin bottom to p tags but its changing styles in other examples too 
![image](https://github.com/user-attachments/assets/39f7dd72-366f-4e22-b103-f7e06be46fe1)
So I've applied margin bottom like this. Can you tell me if this is alright?
```
 &__text:has(+ .ons-btn) {
        margin-bottom: 2rem;
    
```}

Reply:
I don't know what caused this but Im assuming this has been fixed now because the VR tests are passing and theres only the images for the new example been added. This is right, these changes shouldn't affect other components